x-pack/plugins/apm/server/lib/search_strategies/correlations/utils/math_utils.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { HistogramItem } from '../query_ranges';
+import { roundToDecimalPlace } from '../../../../../common/search_strategies/correlations/formatting_utils';
+
+export function* range(start: number, end: number, step: number) {
+  while (start < end) {
+    yield start;
+    start += step;
+  }
+}
```

Comment:
Any advantage to this over lodash `_.range(start, end, step)` ?

Reply:
Switched to lodash's `range` here https://github.com/elastic/kibana/pull/99905/commits/288f394022ea567d3d0953afb4d1a84d9c68abfe